src/tokens/Assets.tsx

```diff
@@ -306,10 +414,16 @@ function AssetsScreen({ navigation, route }: Props) {
       </Animated.View>
 
       <AnimatedSectionList
-        contentContainerStyle={{
-          paddingBottom: insets.bottom,
-          opacity: listHeaderHeight > 0 ? 1 : 0,
-        }}
+        contentContainerStyle={[
+          {
+            paddingBottom: insets.bottom,
+            opacity: listHeaderHeight > 0 ? 1 : 0,
+          },
+          activeTab === AssetTabType.Collectibles && styles.nftsContentContainer,
+          activeTab === AssetTabType.Collectibles && nftsError
+            ? { alignItems: 'center' }
+            : { paddingLeft: Spacing.Regular16 },
```

Comment:
Same as above comment, this should be 24, maybe this should go in nftsContentContainer, since the noNftsText also requires 24 margin instead of 16